postgres/server/init.sls

```diff
@@ -176,6 +169,15 @@ postgresql-pg_hba:
 {%- endif %}
     - require:
       - file: postgresql-config-dir
+    - watch_in:
+      - module: postgresql-service-restart
```

Comment:
I've found that simply reloading the postgres service is not enough to pick up ACL changes in pg_hba.conf. A full restart is required.

This is on ubuntu 18.04 with postgres 10.